spring-cloud-gcp-samples/spring-cloud-gcp-data-firestore-sample/src/main/java/com/example/User.java

```diff
@@ -33,12 +37,16 @@
 
 	int age;
 
+	List<Pet> pets;
```

Comment:
Might be helpful to add an extra comment here to draw attention to it, like:

```
// This represents a Firestore child collection
List<Pet> pets;
```

Reply:
I think that should be clear from the context.